src/main/java/com/school/project/model/entities/Attendance.java

```diff
@@ -1,10 +1,34 @@
 package com.school.project.model.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "ATTENDANCE")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 public class Attendance {
 
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
+    @OneToOne
+    @JoinColumn(name = "LESSON_ID",nullable = false)
```

Comment:
please use Intellij auto styling:

The auto-formatting shortcut in IntelliJ

For windows Ctrl+Alt+L.
For ubuntu Ctrl+Alt+windows+L.
For Mac Command+Option+L.